ibm_catalog.json

```diff
@@ -0,0 +1,346 @@
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager-custom-credentials-engine/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-secrets-manager-custom-credentials-engine/main/images/secrets_manager_custom_credentials_engine.svg",
+      "provider_name": "IBM",
+      "features": [
+        {
+          "title": "IAM service authorization",
+          "description": "Creates a [service authorization policy](https://cloud.ibm.com/docs/account?topic=account-serviceauth&interface=ui) between secrets manager as source and code engine project as target."
+        },
+        {
+          "title": "IAM credentials secret",
+          "description": "Creates an [IAM credentials secret](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-iam-credentials&interface=terraform) to allow code engine job to access/update secrets in secrets manager instance."
+        },
+        {
+          "title": "Custom credentials engine",
+          "description": "Creates a [custom credentials engine](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-custom-credentials-config&interface=terraform) configuration which acts as a back-end for custom credentials secret."
+        }
+      ],
+      "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [here](https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager-custom-credentials-engine/issues). Please note this product is not supported via the IBM Cloud Support Center.",
+      "flavors": [
+        {
+          "label": "Fully configurable",
+          "name": "fully-configurable",
+          "index": 1,
+          "install_type": "fullstack",
+          "working_directory": "solutions/fully-configurable",
+          "compliance": {
```

Comment:
Did you check if there are actually any compliance control that are applicable in this DA? If not, there is no point in adding a compliance section

Reply:
no controls applicable - remove it
<img width="926" height="230" alt="image" src="https://github.com/user-attachments/assets/36b95eee-a90b-43b7-a8d1-6a250f838d38" />